plugins/quickicon/eos/src/Extension/Eos.php

```diff
@@ -0,0 +1,347 @@
+<?php
+
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Quickicon.eos
+ *
+ * @copyright   (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Plugin\Quickicon\Eos\Extension;
+
+use Exception;
+use Joomla\CMS\Access\Exception\NotAllowed;
+use Joomla\CMS\Cache\CacheController;
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Plugin\CMSPlugin;
+use Joomla\Database\DatabaseAwareTrait;
+use Joomla\Event\SubscriberInterface;
+use Joomla\Module\Quickicon\Administrator\Event\QuickIconsEvent;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * Joomla! end of support notification plugin
+ *
+ * @since __DEPLOY_VERSION__
+ */
+final class Eos extends CMSPlugin implements SubscriberInterface
+{
+    use DatabaseAwareTrait;
+
+    /**
+     * The EOS date for 4.4. and beyond
+     *
+     * @var    string
+     * @since __DEPLOY_VERSION__
+     */
+    public const EOS_DATE = '2023-10-25';
+
+    /**
+     * Load the language file on instantiation.
+     *
+     * @var    bool
+     * @since __DEPLOY_VERSION__
```

Comment:
Move `__DEPLOY_VERSION__` up (here 1 space) for a nice alignment. This also applies to others.

```suggestion
     * @var    bool
     * @since  __DEPLOY_VERSION__
```